### PR TITLE
Move settings exe to base of build archive

### PR DESCRIPTION
**Pull request by cartmouse**
## Summary

Move settings exe to base of build archive rather than keeping it in three layers of directories.

## Contributor Tasks

_These tasks are for the pull request creator to tick off._

- [ ] Relevant changelogs have been updated with user-visible changes
    - [ ] [TouchFree Windows](/ultraleap/touchfree/blob/-/CHANGELOG-windows.md)
    - [ ] [TouchFree BrightSign](/ultraleap/touchfree/blob/-/CHANGELOG-brightsign.md)
    - [ ] [TouchFree Web Tooling](/ultraleap/touchfree/blob/-/TF_Tooling_Web/CHANGELOG.md)

## Reviewer Tasks

_Add any instructions or tasks for the reviewer such as specific test considerations before this can be merged._

- [x] Developer testing
- [x] Code reviewed
- [ ] Non-code assets reviewed
- [ ] Documentation reviewed - includes checking documentation requirements are met and not missing e.g., public API is commented

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -397,7 +398,7 @@ TF_Settings::Web::build:
     name: "TouchFree_Settings_Web_${env:TOUCHFREE_VERSION}_${env:CI_COMMIT_SHORT_SHA}"
     paths:
     - ./TF_Settings_Web/dist/*
-    - ./TF_Settings_Web/src-tauri/target/release/*.exe
+    - ./TF_Settings_Web/touchfree-settings-web.exe
```
Comment (GabrielHomewoodUL) [inline .gitlab-ci.yml]: I would say just pull any .exe files from here rather than this named file specifically as we are planning on renaming

Comment (JordanBerni-UL) [reply .gitlab-ci.yml]: Agree - also failed with this error 
`file 'c:\builds\apps\screen-control/TF_Settings_Web/src-tauri/target/release/touchfree-settings-web.exe' does not exist.`

Comment (JordanBerni-UL) [reply .gitlab-ci.yml]: Due to the azuresigntool

Comment (cartmouse) [reply .gitlab-ci.yml]: This should all be sorted now!

Comment (JordanBerni-UL) [reply .gitlab-ci.yml]: Missing the entire exe now 😨

**Review by JordanBerni-UL — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (JordanBerni-UL) [reply .gitlab-ci.yml]: Resolved on my end

**Pull request merged**